activate.sh

```diff
@@ -36,3 +36,11 @@ You can manage your cluster by opening the web application running on
 port 80 of this node from your browser.
 
 EOF
+
+# Generate root ssh key and share it with velum
+# https://bugzilla.suse.com/show_bug.cgi?id=1030876
+if ! [ -f /root/.ssh/id_rsa ]; then
+  ssh-keygen -b 2048 -f /root/.ssh/id_rsa -t rsa -N ""
+fi
+[ -d /tmp/ssh ] || mkdir /tmp/ssh
+cp /root/.ssh/id_rsa.pub /tmp/ssh
```

Comment:
Our intention is to kill the `activate.sh` scrfipt. We should move the key generation somewhere else, no?